vuepress/docs/next/tutorials/getting-started/gke-install.md

```diff
@@ -2,174 +2,149 @@
-   - change the dbms setting from `embedded` to `postgresql`
-   - set `singleHostName` to EXTERNAL-IP.nip.io, e.g. 11.111.111.111.nip.io. An alternative is to route your own DNS to the EXTERNAL-IP but this setup will work immediately.
+You can now deploy your application to your GKE cluster
 
-6. Run helm to generate the template file:
+1. Deploy the Entando Application 
+```sh
+kubectl apply -n entando -f entando-app.yaml
 ```
-helm template quickstart --namespace=entando ./ > quickstart.yaml
-```
-7. Deploy the Entando Application 
-```
-kubectl apply -n entando -f quickstart.yaml
+2. It can take around 10 minutes for the application to fully deploy. You can watch the pods warming up with the command below. Use `Ctrl+C` to exit.
+```sh
+kubectl get pods -n entando --watch
 ```
 
-9. Watch the deployment for completion
+3. Once all the pods are in a running state, access the Entando App Builder at the following address
 ```
-kubectl -n entando get pods --watch
+http://YOUR-HOST-NAME/app-builder/
 ```
 
-Watch Entando startup. The application will be available when the `quickstart-composite-app-deployer` pod has a status of completed  
-
-```
-NAME                                                 READY   STATUS    RESTARTS   AGE
-entando-operator-5cdf787869-s8bwg                    1/1     Running   0          18m
-quickstart-ab-deployment-5655bc6cc6-g55kr            1/1     Running   0          98s
-quickstart-cm-deployment-6565fc67b8-cwgpc            0/1     Running   0          62s
-quickstart-composite-app-deployer-9419               1/1     Running   0          10m
-quickstart-db-deployment-6fdb96f98c-fgm5g            1/1     Running   0          5m55s
-quickstart-deployer-9708                             1/1     Running   0          6m3s
-quickstart-eci-k8s-svc-deployment-588ffd65c8-p5xcl   1/1     Running   0          7m7s
-quickstart-kc-db-deployment-67c7fd4fbf-scwvj         1/1     Running   0          10m
-quickstart-kc-server-deployment-7dbcc968f8-rg5lc     1/1     Running   0          9m31s
-quickstart-server-deployment-6d966c78c4-tp6cz        1/1     Running   0          3m17s
-```
+See the [Getting Started guide](../../docs/getting-started/README.md#log-in-to-entando) for helpful login instructions and next steps.
 
 ## TLS Notes
 
-If you have configured TLS related to the ingress for the ECI service
+### ECI Service Ingress
 
-1. After the installation patch the ingress path for the ECI to include a trailing slash
+If you have configured TLS for the ECI service ingress, patch the ingress path after installation to include a trailing slash
 
 ```
 kubectl -n {NAMESPACE} patch ingress quickstart-eci-ingress --type='json' -p='[{"op": "replace", "path": "/spec/rules/0/http/paths/0/path", "value":"/k8s/"}]'
 ingress.extensions/quickstart-eci-ingress patched
 ```
 
-## Workaround for invalid login form, redirect when using TLS
+### Workaround
+
+Bypass an invalid login form and redirect when using TLS using this workaround.
```

Comment:
a bit confusing >> Bypass an invalid login form and redirect when using TLS with this workaround 

not sure if this is what was meant